src/pages/about.js

```diff
@@ -0,0 +1,229 @@
+    marginBottom: 'auto'
+  },
+  enrtfText: {
+    minWidth: '60%',
+    flex: 1
+  }
+});
+
+
+const About = ({ data: { enrtfLogo, dynamicUsers, staticUsers } }) => {
+  const classes = useStyles();
+  const staticUserList = staticUsers.data;
+  return (
+    <Layout>
+      <SEO title="About" />
+      <div className={classes.aboutPage}>
+        <div className={classes.aboutCards}>
+
+          <Card>
+            <CardContent>
+              <Typography gutterBottom variant="h5" component="h2">Pollinator Facts</Typography>
+              <Typography>
+                Pollinator Facts website was created alongside the <a href="https://buzzabout.app">Buzz About</a> participatory simulation game to
+                teach people about native pollinators of Minnesota. This website holds many interesting tidbits as well beautiful pictures of pollinators, their food, homes
+                and habits.
+              </Typography>
+            </CardContent>
+
+            <CardActions>
+              <Button startIcon={<GitHubIcon />} href="https://github.com/mn-pollinators/pollinator-facts" >
+                Github Repo
+              </Button>
+            </CardActions>
+          </Card>
+
+          <Card>
+            <CardContent>
+              <Typography gutterBottom variant="h5" component="h2">Minnesota Pollinators</Typography>
+              <Typography>
+                Minnesota Pollinators is an organization of <a href="https://morris.umn.edu/">University of Minnesota Morris</a> students and faculty
+                working on projects that help the native pollinators of Minnesota.
+                This project is part of a <a href="https://wcroc.cfans.umn.edu/restoring-native-prairie">grant-funded project</a> to restore native prairie
+                on a site in Morris, MN and help people learn about pollinators and the native prairie plants that support them.
+              </Typography>
+            </CardContent>
+
+            <CardActions>
+              <Button startIcon={<GitHubIcon />} href="https://github.com/mn-pollinators" >
+                Github
+              </Button>
+            </CardActions>
+          </Card>
+
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2">Contributors</Typography>
+
+              <List component="nav">
+                {dynamicUsers.nodes.map((contributor) => {
+                  const displayName = contributor.name?? contributor.login;
+                  return (
+                    <ListItem button component="a" href={contributor.htmlUrl} key={contributor.login}>
+                      <ListItemAvatar>
+                        <Avatar alt={displayName} src={contributor.avatarUrl} />
+                      </ListItemAvatar>
+                      <ListItemText primary={displayName}/>
+                    </ListItem>
+                  )
+                })}
+
+                {Object.keys(staticUserList).map((objectName, index) => {
```

Comment:
Why not use `Object.values` here?

Reply:
[Object.values()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values#Browser_compatibility) is not supported on Internet Explorer while [.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys#Browser_compatibility) is.

Reply:
That shouldn't be an issue since Gatsby uses Babel and will polyfill newer features when needed.